packages/lib/bookings/filterHostsByLeadThreshold.ts

```diff
@@ -0,0 +1,135 @@
+import prisma from "@calcom/prisma";
```

Comment:
This file is so close to being domain logic only. Can we move the prisma call to retrieve data outside of this and supply the data to these functions instead? Would be great to start building more domain-only code.